lexicons/app/bsky/graph/starterpack.json

```diff
@@ -0,0 +1,50 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.starterpack",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "Record defining a starter pack of actors and feeds for new users.",
+      "key": "any",
+      "record": {
+        "type": "object",
+        "required": ["name", "createdAt"],
+        "properties": {
+          "name": {
+            "type": "string",
+            "maxLength": 64,
+            "minLength": 1,
+            "description": "Display name for starter pack; can not be empty."
+          },
+          "description": {
+            "type": "string",
+            "maxGraphemes": 300,
+            "maxLength": 3000
+          },
+          "descriptionFacets": {
+            "type": "array",
+            "items": { "type": "ref", "ref": "app.bsky.richtext.facet" }
+          },
+          "list": {
+            "type": "string",
+            "format": "at-uri",
+            "description": "Reference (AT-URI) to the list record."
+          },
+          "feeds": {
+            "type": "array",
+            "maxLength": 50,
```

Comment:
Note to self that we want to figure out the right max for this. I'm leaning toward 3 after some more though (so 5 feeds pinned when you sign up including Discover and Following) but will try to get a consensus. Definitely going to be smaller than 50 though 😅